examples/durable-functions/src/functions/hello_world.rs

```diff
@@ -1,17 +1,18 @@
-use azure_functions::{bindings::DurableOrchestrationContext, func};
+use azure_functions::{bindings::DurableOrchestrationContext, durable::OrchestrationOutput, func};
 use futures::future::join_all;
 
 #[func(name = "HelloWorld")]
```

Comment:
Should we just leave this as `#[func]`? If we are using the snake case convention, this might throw people off.

Reply:
Agreed, let me scrub the example.

Reply:
Pushed the fix.